internal/app/machined/pkg/adapters/perf/cpu.go

```diff
@@ -41,8 +40,22 @@ func (a cpu) Update(stat *procfs.Stat) {
 		}
 	}
 
-	translateListOfCPUStat := func(in []procfs.CPUStat) []perf.CPUStat {
-		return slices.Map(in, translateCPUStat)
+	translateListOfCPUStat := func(in map[int64]procfs.CPUStat) []perf.CPUStat {
```

Comment:
Since it's no longer a simple lambda-like function maybe we should promote it and `translateCPUStat` to standalone global functions?

Reply:
no, it's different actually in both places - it translates to different data structures

Reply:
Ah, okay!